### PR TITLE
Add a highlight tag, Cleanup.

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

File: site/_docs/plugins.md [hunk 1]
```diff
@@ -31,22 +31,26 @@ Jekyll generates your site.
 2. In your `_config.yml` file, add a new array with the key `gems` and the
 values of the gem names of the plugins you'd like to use. An example:
 
+{% highlight text %}
```
Comment (pathawks) [inline site/_docs/plugins.md]: Why not `yaml`?

Comment (chrisfinazzo) [inline site/_docs/plugins.md]: You're right, that's a better choice.

Comment (envygeeks) [inline site/_docs/plugins.md]: We can just fix our site CSS and start using <code>```</code> with Pygments, I honestly don't know why we haven't before now.  Any reason @parkr? Considering Github fully supports Pygments with <code>```</code>

**Comment by parkr** [comment]
I'm not sure this will render the way we want. Can you provide screenshots of before and after?

**Comment by chrisfinazzo** [comment]
Before: 

screen shot 2015-11-25 at 2 20 21 pm

After: 

screen shot 2015-11-25 at 2 17 53 pm

**Comment by parkr** [comment]
Hm... item 3 was numbered as item 1. Not as we want them to be. Can we achieve these results while keeping the numbers properly aligned?

**Comment by super-ala** [comment]
sorry  masters      i  just  binging   use   this    and  dont  know
   please   tell  me   how  can   i   look  my   last  gf  bace   book
        and  dont  let  she  konw >!??!!!thanks  !!!!!!

2015-11-26 3:59 GMT+08:00, Parker Moore <notifications@github.com>:

**Comment by rebornix** [comment]
@parkr , you may want to take a look at #4121 . The highlight issue @chrisfinazzo mentioned here is caused by  GitHub multiline code block syntax, we don't have any background CSS for it. In my PR, I use CSS to handle it instead of content change, as we have a lot of code block still using GitHub syntax. 

Jekyll team's other members have different opinions on how we style long single line code block, the PR is still pending.

**Comment by chrisfinazzo** [comment]
@rebornix Now that you mention it, I'm not surprised. I spent some this afternoon deconstructing this and  trying to figure out what was happening, only to end up essentially where your PR is. 

Incidentally, I think we may be stepping on each other's toes with the changes here. I'll leave it to the Core guys to decide which fixes they want to pick out.

**Pull request closed without merging**